README.md

```diff
@@ -45,23 +45,67 @@ yarn add @syndicateio/syndicate-node
 
 [![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/edit/syndicate-typescript?file=app.ts&view=editor)
 
-```typescript
+
+### Send a transaction
+```js
 import { SyndicateClient } from "@syndicateio/syndicate-node";
 
 const syndicate = new SyndicateClient({ token: 'YOUR_ACCESS_TOKEN' })
 
 const tx = await syndicate.transact.sendTransaction({
-  projectId: 'eb7dab54-f66b-4739-8c18-f5a6196f7117',
-  contractAddress: '0x4Bc7a03C46c0BBb290d5455ff7f03d30263809e4',
-  chainId: 80001,
-  functionSignature: 'mintTo(address to)',
+  projectId: 'YOUR_ACCESS_TOKEN',
+  contractAddress: "0x52962dd492dDDef76d4eFb2bB7E505aeAE4554A1",
+  chainId: 84532,
+  functionSignature: "mint(address account)",
   args: {
-    to: '0x18F33CEf45817C428d98C4E188A770191fDD4B79'
-  },
+    account: "0x8A05fA58d533a6e40C4381E3247Cf4c68ca61cdc"
+  }
 })
 console.log('Transaction ID Received:', tx.transactionId)
 ```
 
+### Get transaction hash from a transaction ID
+```typescript
+import { SyndicateClient } from "@syndicateio/syndicate-node";
+
+interface GetTxhashFromIdOptions {
+  projectID: string
+  txID: string
+  maxAttempts?: number
+  every?: number
+}
+
+export async function getTxHashFromTxId(client: SyndicateClient, {
```

Comment:
Do we want to show them this code or just add in a helper in our node library?

Reply:
so i chatted with @calebguy about this, we want to add as a helper but can't yet cuz we're running into an issue with fern, he reached out to them. was thinking we could merge this in the meantime, and once we are able to update it we can change the readme @sammdec

Reply:
Gotcha! Yeah totally makes sense let's spin up a ticket to remind ourselves to add it in later

Reply:
here we go cc @calebguy  https://linear.app/syndicate/issue/ENG-6467/add-wait-on-tx-hash-function-to-node-sdk-utils